sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -306,7 +306,9 @@ public abstract static class Read extends PTransform<PBegin, PCollection<Row>> {
     /**
      * Returns the Google Cloud Bigtable instance being read from, and other parameters.
      *
-     * @deprecated please use {@link #getBigtableReadOptions()}.
+     * @deprecated please set the configurations directly:
+     *     BigtableIO.read().withProjectId(projectId).withInstanceId(instanceId).withTableId(tableId)
```

Comment:
What does this change mean? Is `getBigtableReadOptions()` no longer recommended also? The original `deprecated` message was clear and now it seems talking about another thing (set, instead of get configuration)

Reply:
Ah thanks, my bad..  However, BigtableReadOptions and BigtableWriteOptions are actually package private. Going forward we want customers to configure these options directly on BigtableIO.read() or write(), so there won't be one getter that'll show all the configuration. I updated the doc to use populateDisplayData instead, do you think this will work?